app/src/main/java/org/mozilla/tv/firefox/onboarding/ReceiveTabPreboardingActivity.kt

```diff
@@ -42,6 +42,7 @@ class ReceiveTabPreboardingActivity : AppCompatActivity() {
 
         buttonNotNow.setOnClickListener {
             finish()
+            TelemetryIntegration.INSTANCE.fxaPreboardingDismissButtonClickEvent()
```

Comment:
I spoke with athomasmoz & the data folks: **we want to rename the probe**. The recommendation was that the data owner should decide but the recommendation (for this specific case) from the data team is to rename it because it will make analysis easier.

Can you rename the probe and update the docs, including mentioning the old probe is deprecated and linking to the issue with the bug?

Reply:
Good call!  Will do.